packages/cli/src/dbt/context.ts

```diff
@@ -15,16 +16,33 @@ export type DbtContext = {
 
 export const getDbtContext = async ({
     projectDir,
+    initialProjectDir,
 }: GetDbtContextArgs): Promise<DbtContext> => {
     const projectFilename = path.join(projectDir, 'dbt_project.yml');
     let config;
+
     try {
         config = yaml.load(
             await fs.readFile(projectFilename, { encoding: 'utf-8' }),
         ) as any;
-    } catch (e) {
+    } catch (e: any) {
+        if (projectDir !== '/') {
```

Comment:
This isn't platform dependent. I think `os` or `path` will give you an option to check if you're at the root directory